### PR TITLE
DOC: Include instructions for starting a Redis server and worker.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,14 +37,25 @@ It’s a Rails app that:
...
 
     ```sh
     $ bundle exec rails server
     ```
 
     You should now have a server running and can visit it at http://localhost:3000/. Open that up in a browser and go to town!
 
+9. Bulk importing (and, in the future, potentially other features) make use of a
+   Redis queue. If you plan to use any of these features, you must also start a
+   Redis server and worker.
```
Comment (Mr0grog) [inline README.md]: Might be helpful to include basic install instructions (e.g. `brew install redis` on OS X, `apt-get install redis` on Debians) and/or a link to https://redis.io for people who don’t know Redis.

File: README.md [hunk 2]
```diff
@@ -37,14 +37,25 @@ It’s a Rails app that:
...
+
+   ```sh
+   $ redis-server
+   ```
+
+   ```sh
+   $ QUEUE=* VERBOSE=1 bundle exec rake environment resque:work
```
Comment (Mr0grog) [inline README.md]: I think it would be helpful to title these two lines of code, e.g. “start Redis” and “start the worker”

File: README.md (hunk 1 above)
Comment (danielballan) [inline README.md]: addressed above at "Ensure you have Redis"

**Comment by danielballan** [comment]
Good suggestions, thanks!

**Comment by Mr0grog** [comment]
👍 Beautiful!

**Pull request merged**